### DISCUSSION TITLE
wikiimage_sub and sub preview don't include attached shuttles

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#10095] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

wikiiimage_sub command and sub previews (from save window, from the submarine overview) don't include any attached shuttles

### Reproduction steps

1. load kastrull
2. make wikiiimage_sub command / save preview / go into test mode and open submarine view
3. where shuttle? (drone)

### Bug prevalence

Happens every time I play

### Version

0.19.11.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Ek-Videogames** [comment]
For image compositing reasons it might be useful to have some way to turn this on or off, maybe like an optional argument.
